datasets/wikiann/wikiann.py

```diff
@@ -43,7 +43,7 @@
 _DESCRIPTION = """WikiANN (sometimes called PAN-X) is a multilingual named entity recognition dataset consisting of Wikipedia articles annotated with LOC (location), PER (person), and ORG (organisation) tags in the IOB2 format. This version corresponds to the balanced train, dev, and test splits of Rahimi et al. (2019), which supports 176 of the 282 languages from the original WikiANN corpus."""
 
 # use ?dl=1 to force browser to download data instead of displaying it
-_DATA_URL = "https://www.dropbox.com/s/12h3qqog6q4bjve/panx_dataset.tar?dl=1"
+_DATA_URL = "https://s3.amazonaws.com/datasets.huggingface.co/wikiann/1.1.0/panx_dataset.zip"
```

Comment:
i guess you can then remove the comment line above this one

Reply:
Yes, you are right @julien-c 😅